package.json

```diff
@@ -50,7 +50,7 @@
     "vl-ui-button": "^5.0.10",
     "vl-ui-datepicker": "^3.2.5",
     "vl-ui-form-grid": "^3.0.9",
-    "vl-ui-form-message": "^5.0.7",
+    "vl-ui-form-message": "milieuinfo/webcomponent-vl-ui-form-message#formValidationMessageIsHiddenAlsHetErrorIdAttribuutHeeft",
```

Comment:
mag dit zo naar master

Reply:
dit moet nog aangepast worden als de nieue release klaar is.